### ISSUE TITLE
Bad indentation in haml does not trigger error

### ISSUE THREAD

**Issue by tongrhj** [adrienverge/yamllint#70]
`sample.yml`:
```
pt-BR:
  emails:
    verification:
      body:         |
        :plain
          Oi #{contact.first_name},

         This line should give an error #{variable_name}.

          Clique no link abaixo para ativar sua conta.
```

Expected: `error indentation expected 10 got 9`

Instead: no error, running yamllint v1.8.1 on default configuration

**Comment by adrienverge** [comment]
Hi,

Running yamllint on your input gives me:
```
sample.yaml
  1:1       warning  missing document start "---"  (document-start)
  4:20      error    too many spaces after colon  (colons)
  6:11      error    wrong indentation: expected 8 but found 10  (indentation)
  8:10      error    wrong indentation: expected 8 but found 9  (indentation)
  10:11     error    wrong indentation: expected 8 but found 10  (indentation)
```

which looks perfectly normal to me, as:
```
:plain
  Oi #{contact.first_name},

 This line should give an error #{variable_name}.

  Clique no link abaixo para ativar sua conta.
```
is a multi-line string stored in the `pt-BR.emails.verification.body` property.

I don't know what `:plain` is, but it is not a YAML directive. If you want the linter to pass with `check-multi-line-strings: true` in conf, you need to write:
```
:plain
Oi #{contact.first_name},

This line should give an error #{variable_name}.

Clique no link abaixo para ativar sua conta.
```

You can also [disable `check-multi-line-strings`](https://yamllint.readthedocs.io/en/latest/rules.html#module-yamllint.rules.indentation) if you prefer.
